server/playerServer/playerserver.js

```diff
@@ -451,6 +457,12 @@ var server = http.createServer(function (req, res) {
 			}));
 			return;
 		}
+		if( x !== x || y !== y){
```

Comment:
Using isNaN is clearer in these cases.
Example isNaN(x) || isNaN(y)
This way other people understand better what is going on.